nautobot/core/templatetags/helpers.py

```diff
@@ -393,17 +393,19 @@ def humanize_speed(speed):
         1544 => "1.544 Mbps"
         100000 => "100 Mbps"
         10000000 => "10 Gbps"
+        1000000000 => "1 Tbs"
+        1600000000 => "1.6 Tbps"
+        10000000000 => "10 Tbps"
     """
     if not speed:
         return ""
-    if speed >= 1000000000 and speed % 1000000000 == 0:
-        return f"{int(speed / 1000000000)} Tbps"
-    elif speed >= 1000000 and speed % 1000000 == 0:
```

Comment:
Probably worth adding a `.changed` fragment for the change to `humanize_speed` for decimal Gbps and Tbps speeds?